packages/common/src/errors.js

```diff
@@ -1,6 +1,41 @@
-import { createError } from 'apollo-errors';
+// @flow
 
-const AccountsError = createError('AccountsError', {});
+import type { PasswordLoginUserType } from './types';
 
 // eslint-disable-next-line import/prefer-default-export
-export { AccountsError };
+export class AccountsError extends Error {
+  loginInfo: PasswordLoginUserType;
+  errorCode: string | number;
+  epochTime: number;
+
+  constructor(message: string, loginInfo?: PasswordLoginUserType, errorCode?: string | number) {
```

Comment:
Actually I would like this parameter to be only the login info for later use in the audit system. We can add in additionalData field for other things as well, what kind of data are you thinking of sending? 
Also, keep in mind that you always use string templates to add in relevant data to the error message. `message` filed will actually get picked up by any console or error handler like react-native's "red screen" and should actually help with fixing bugs when using this package.